tests/test_cityTemporalTiler.py

```diff
@@ -22,7 +22,7 @@ class Test_Tile(unittest.TestCase):
     def test_temporal(self):
         city_temp_tiler = CityTemporalTiler()
         output_dir = Path("tests/city_temporal_tiler_test_data/junk/temporal")
```

Comment:
I know I might be the original culprit, but maybe we could find better than "junk" ?

Reply:
It would be nice indeed. There is some other output directories (especially in tests) that should be renamed. I'll add it to my todo list and make those small changes later (not in this PR).

Reply:
In [this commit](https://github.com/VCityTeam/py3dtilers/commit/7ff750d82d6841c45d09d3ecc6786b76954ef406) (on master), I renamed some test output folders. Now, all the tilesets generated by the tests are in folders called `generated_tilesets`